src/neuroconv/datainterfaces/ecephys/blackrock/blackrockdatainterface.py

```diff
@@ -19,6 +19,8 @@
 class BlackrockRecordingExtractorInterface(BaseRecordingExtractorInterface):
     """Primary data interface class for converting a BlackrockRecordingExtractor."""
 
+    RXName = "BlackrockRecordingExtractor"
```

Comment:
This is an alternative demonstration that maintains the current name of the extractor and merely corrects it to match SpikeInterface. Though something like this could also just be added to the default auto-naming rule, like `__name__.strip("Interface") if "Extractor" in __name__ else __name__.replace("Interface", "Extractor")` or something...